### DISCUSSION TITLE
🚀 Feature request: Native Date String Translations

### DISCUSSION THREAD

**Discussion by bsrweb** [ExpressionEngine/ExpressionEngine#4270] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**

This may be covered/redundant with this translation discussion? https://github.com/ExpressionEngine/ExpressionEngine/discussions/2930

If so, please delete/close.

For years, EE's only solution for translating date strings was a third party plug called, Hippo DateTime. Sadly, the developer passed away and the plugin is no longer compatible with the latest version of PHP. I think the date function currently used in the plugin is deprecated and I paid a developer to modify our paid copy of the plugin to use Carbon PHP library.

Given that there's no longer any support for this particular feature and i18l is a rather important feature in a CMS, I wanted to bring this issue up.

**Describe the solution you'd like**

As above...native support for translating date strings, leveraging the Carbon library.

**Describe alternatives you've considered**

As above...paying a developer to update an existing, but no longer maintained, third party plugin.

**Teachability, Documentation, Adoption, Migration Strategy**

Perhaps extend the native date field tags? Something like, `{entry_date:convert isoformat="YYYY年M月D日" locale="ja_JP.UTF-8}`, using the parameters supported in the Carbon library.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
This is also already being requested in https://github.com/ExpressionEngine/ExpressionEngine/discussions/3579

Also a PR: https://github.com/ExpressionEngine/ExpressionEngine/pull/4669
